biome.json

```diff
@@ -42,10 +52,21 @@
 	},
 	"overrides": [
 		{
-			"include": ["package.json"],
+			"includes": ["**/*.astro"],
+			"linter": {
+				"rules": {
+					"correctness": {
+						"noUnusedVariables": "off",
+						"noUnusedImports": "off"
+					}
+				}
+			}
+		},
+		{
+			"includes": ["**/package.json"],
 			"json": {
 				"formatter": {
-					"lineWidth": 1
+					"indentStyle": "space"
```

Comment:
What’s the motivation for using spaces for JSON files? (I noticed we’re reformatting `package.json` in this PR from tabs to spaces.)

Reply:
Because when the file is updated via pnpm, it uses spaces. I can revert that if you want